CHANGES.rst

```diff
@@ -4,6 +4,7 @@
 Enhancements and Fixes
 ----------------------
 
+- Make deletion of TAP jobs optional via a delete flag. Default: True
```

Comment:
We track the PR's number in the changelog, thus the CI check was failing for this.
```suggestion
- Make deletion of TAP jobs optional via a new ``delete`` kwarg. [#640]
```